sdk/confidentialledger/azure-confidentialledger/swagger/README.md

```diff
@@ -23,7 +23,7 @@ python: true
 #### Tag: identity-service
 These settings apply only when `--tag=identity-service` is specified on the command line.
 ```yaml $(tag) == 'identity-service'
-input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/identityservice.json
+input-file: https://github.com/Azure/azure-rest-api-specs/blob/main/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/preview/2024-08-22-preview/identityservice.json
```

Comment:
I can't comment on the line but on L43-44 in this file, flip the order of the tags so the identity-service generates before confidential-ledger:

```
batch:
  - tag: identity-service
  - tag: confidential-ledger
```

Then retry the regeneration command. I think this should stop the azure/confidentialledger/__init__.py file from getting overwritten and making the client un-importable.